compiler/encoders/asm/asmSemScript.sml

```diff
@@ -3,6 +3,7 @@
 *)
 open HolKernel Parse boolLib bossLib
 open asmTheory machine_ieeeTheory
+open miscTheory (* for bytes_in_memory *)
```

Comment:
Surely should be an `open preamble` here.

Reply:
I agree, but I think we'll have to take that up with @myreen.

Reply:
Absent @myreen's input, this stays as-is for now.